solar/dblayer/riak_client.py

```diff
@@ -30,9 +30,9 @@ def session_end(self, result=True):
     def delete_all(self, cls):
         for _ in xrange(10):
             # riak dislikes deletes without dvv
-            rst = cls.bucket.get_index('$bucket',
-                                       startkey='_',
-                                       max_results=100000).results
+            # bitcast doesnt support indexes, and we need
+            # to use mutlibackends
+            rst = cls.bucket.get_keys()
```

Comment:
the problem is that with 2i enabled backend this is very slow, and very time consuming. We should check type there and use 2i thing when 2i is there.